loom/graphics/gfxVectorRenderer.cpp

```diff
@@ -705,7 +705,8 @@ void VectorRenderer::reset()
 
 void VectorRenderer::initialize()
 {
-    nvgSetAllocFunctions(nvgAlloc, nvgRealloc, nvgFree);
+    nvgSetAllocFunctions(customAlloc, customRealloc, customFree);
+    nsvgSetAllocFunctions(customAlloc, customRealloc, customFree);

```

Comment:
I was going to complain about this line but I see now it's svg vs vg. Good catch.